contracts/amm/Milton.sol

```diff
@@ -803,6 +803,20 @@ abstract contract Milton is MiltonInternal, IMilton {
                 transferAmount,
                 decimals
             );
+            uint256 wadMiltonErc20BalanceBeforeRedeem = IERC20Upgradeable(_asset).balanceOf(address(this));
+            if (wadMiltonErc20BalanceBeforeRedeem <= transferAmountAssetDecimals) {
+                IporTypes.MiltonBalancesMemory memory balance = _getAccruedBalance();
+                int256 rebalanceAmount = IJoseph(_joseph).calculateRebalanceAmountBeforeWithdraw(
+                    wadMiltonErc20BalanceBeforeRedeem,
+                    balance.vault,
+                    transferAmount + liquidationDepositAmount
```

Comment:
liquidationDepositAmount is necessary here?

Reply:
It prevents us from the situation if will be not enough assets to transfer to the liquidator.